scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,1053 @@
+    cols_dict = _get_mapper_cols_dict("header_mappers")
+    for idx, val in enumerate(cols):
+        cols[idx] = cols_dict.get(val, val)
+    df.columns = cols
+    df["STATE"] = df["STATE"].astype("str").str.pad(width=2,
+                                                    side="left",
+                                                    fillchar="0")
+    df["COUNTY"] = df["COUNTY"].astype("str").str.pad(width=3,
+                                                      side="left",
+                                                      fillchar="0")
+    df["Location"] = "geoId/" + df["STATE"] + df["COUNTY"]
+    # Deriving New Columns
+    derived_cols = _get_mapper_cols_dict("county_2000_2009")
+    df = _derive_cols(df, derived_cols)
+    cols = cols + list(derived_cols.keys())
+    df["Age"] = df["AGEGRP"].apply(_get_age_grp_county_2000_2009)
+    f_cols = [val for val in cols if "Hispanic" in val]
+    df = _unpivot_df(df, ["Year", "Location", "Age"], f_cols)
+    # Creating SV's name using SV, Age Column
+    df["SV"] = df.apply(lambda row: _create_sv(row.SV, row.Age), axis=1)
+    # Deriving Measurement Method for the SV's
+    f_cols = ["Year", "Location", "SV", "Measurement_Method", "Count_Person"]
+    df = _add_measurement_method(df, "SV", "Measurement_Method")
+    return df[f_cols]
+
+
+def _process_county_2010_2020(file_path: str) -> pd.DataFrame():
+    """
+    Returns the Cleaned DataFrame consists
+    county data for the year 2010-2020.
+
+    Args:
+        file_path (str): Input File Path
+
+    Returns:
+        pd.DataFrame: Cleaned DataFrame
+    """
+
+    df = _load_df(file_path, "csv", header=0, encoding='latin-1')
+    df = df[(~df["YEAR"].isin([1, 2, 13])) &
+            (df["AGEGRP"] != 0)].reset_index(drop=True)
+    df["YEAR"] = df["YEAR"].astype('str').str.replace("14", "13").astype("int")
+    df["YEAR"] = 2007 + df["YEAR"]
+    # Mapping Dataset Headers to its FullForm
+    cols_dict = _get_mapper_cols_dict("header_mappers")
+    cols = df.columns.to_list()
+    for idx, val in enumerate(cols):
+        cols[idx] = cols_dict.get(val, val)
+    df.columns = cols
+    df["STATE"] = df["STATE"].astype("str").str.pad(width=2,
+                                                    side="left",
+                                                    fillchar="0")
+    df["COUNTY"] = df["COUNTY"].astype("str").str.pad(width=3,
+                                                      side="left",
+                                                      fillchar="0")
+    df["Location"] = "geoId/" + df["STATE"] + df["COUNTY"]
+    # Deriving New Columns
+    derived_cols = _get_mapper_cols_dict("county_2010_2020")
+    df = _derive_cols(df, derived_cols)
+    cols = cols + list(derived_cols.keys())
+    f_cols = [val for val in cols if "Hispanic" in val]
+    df["Age"] = df["AGEGRP"].apply(_get_age_grp)
+    df["Age"] = df["Age"].str.replace("85To89", "85")
+    df = _unpivot_df(df, ["Year", "Location", "Age"], f_cols)
```

Comment:
This just looks like wrapper function replaceing standard function (just 1 line code for standard function call) and not much helpful. Any such helper methods please remove it.